scripts/ci/changelog/bin/changelog

```diff
@@ -84,9 +84,9 @@ end
 
 POLKADOT_COLLECTIVES_DIGEST = ENV['COLLECTIVES_POLKADOT_DIGEST'] || 'digests/collectives-polkadot-srtool-digest.json'
 SHELL_DIGEST = ENV['SHELL_DIGEST'] || 'digests/shell-srtool-digest.json'
-ASSET_HUB_WESTEND_DIGEST = ENV['ASSET_HUB_WESTEND_DIGEST'] || 'digests/westmint-srtool-digest.json'
-ASSET_HUB_KUSAMA_DIGEST = ENV['ASSET_HUB_KUSAMA_DIGEST'] || 'digests/statemine-srtool-digest.json'
-ASSET_HUB_POLKADOT_DIGEST = ENV['ASSET_HUB_POLKADOT_DIGEST'] || 'digests/statemint-srtool-digest.json'
+ASSET_HUB_WESTEND_DIGEST = ENV['ASSET_HUB_WESTEND_DIGEST'] || 'digests/asset-hub-westend-srtool-digest.json'
+ASSET_HUB_KUSAMA_DIGEST = ENV['ASSET_HUB_KUSAMA_DIGEST'] || 'digests/asset-hub-kusama-srtool-digest.json'
+ASSET_HUB_POLKADOT_DIGEST = ENV['ASSET_HUB_POLKADOT_DIGEST'] || 'digests/asset-hub-westend-srtool-digest.json'
```

Comment:
I suppose you checked that those files are created as such ? (ie `asset-hub-westend-srtool-digest.json`).

Reply:
yep, new files look like this:
![Screenshot 2023-06-26 at 13 16 58](https://github.com/paritytech/cumulus/assets/6053182/60e5a053-0305-48b1-b85b-1dcafccd4017)